apps/ledger-live-mobile/src/screens/Manager/index.tsx

```diff
@@ -1,68 +1,28 @@
-import React, { Component } from "react";
+import React, { useEffect, useState } from "react";
 import { StyleSheet } from "react-native";
-import { useIsFocused } from "@react-navigation/native";
-import { useDispatch } from "react-redux";
+import {
+  useIsFocused,
+  useNavigation,
+  useRoute,
+} from "@react-navigation/native";
 import { Trans } from "react-i18next";
-import manager from "@ledgerhq/live-common/manager/index";
-import { disconnect } from "@ledgerhq/live-common/hw/index";
 import { Device } from "@ledgerhq/live-common/hw/actions/types";
 
 import connectManager from "@ledgerhq/live-common/hw/connectManager";
 import { createAction } from "@ledgerhq/live-common/hw/actions/manager";
-import { Text } from "@ledgerhq/native-ui";
-import { removeKnownDevice } from "../../actions/ble";
+import { Flex, Text } from "@ledgerhq/native-ui";
 import { ScreenName } from "../../const";
 import { ManagerTab } from "./Manager";
-import SelectDevice from "../../components/SelectDevice";
+import SelectDevice from "../../components/SelectDevice2";
```

Comment:
If we are replacing it, can't we drop SelectDevice in favor of SelectDevice2 or is it still in use elsewhere?

Reply:
We're not replacing it just yet. I've added added a feature flag to control the usage of the new one. This way we can merge the code and work on the few issues that are left on another PR.

Reply:
Can we rename it to NewSelectDevice or something? The name like this looks like a test name or an error